learn/core_concepts/documents.md

```diff
@@ -54,12 +54,35 @@ An attribute functions a bit like a variable in most programming languages, i.e.
 
 Every field has a [data type](/reference/under_the_hood/datatypes.md) dictated by its value. Every value must be a valid [`JSON` data type](https://www.w3schools.com/js/js_json_datatypes.asp).
 
-Take note that in the case of strings, the value **[can contain at most 1000 words](/reference/features/known_limitations.md#maximum-words-per-attribute)**. If it contains more than 1000 words, only the first 1000 will be indexed.
-
 You can also apply [<clientGlossary word="ranking rules" />](/learn/core_concepts/relevancy.md#ranking-rules) to some fields. For example, you may decide recent movies should be more relevant than older ones.
 
 If you would like to adjust how a field gets handled by MeiliSearch, you can do so in the [settings](/reference/features/settings.md#settings).
 
+Take note that in the case of strings, the value **[can contain at most 1000 positions](/reference/features/known_limitations.md#maximum-words-per-attribute)**. Words exceeding the 1000 position limit will be ignored.
+
+### Example
+
+If your query is `Hello World`
+
+- `Hello` takes the position `0` of the attribute
+- `World` takes the position `1` of the attribute
+
+If your query is `Hello, World`
+
+- `Hello` takes the position `0` of the attribute
+- `,` takes the position `8` of the attribute
+- `World` takes the position `9` of the attribute
+
+`,` takes 8 spaces as it is a hard separator, you can read more about it [here](https://docs.meilisearch.com/reference/under_the_hood/datatypes.html#string).
```

Comment:
What do you think about having this inside a `note` container? Is it too much?

I would separate these two clauses into two separate sentences as the comma feels too colloquial here.

Also, personal preference: I try to  avoid using `here` as the text of a link because it gives the reader very little information on what they're going to find. Not super important in this case, though.

What do you think about this:

```suggestion
::: note
`,` takes 8 spaces as it is a hard separator. You can read more about word separators in our [article about datatypes](https://docs.meilisearch.com/reference/under_the_hood/datatypes.html#string).
:::
```

Reply:
Why didn't I think of that, yeah the note container is a good idea
Will keep the here thing in mind for future changes